website/docs/cli/commands/describe/describe-component.mdx

```diff
@@ -55,6 +55,8 @@ atmos describe component vpc -s tenant1-ue2-dev --skip=include,eval
 atmos describe component vpc -s plat-ue2-prod --query .vars.tags
 
 atmos describe component vpc -s plat-ue2-prod -q .settings
+
+atmos describe component vpc -s plat-ue2-prod --pager=more
```

Comment:
`--pager=more` doesn't show the pager, I see `less` and `yes` works. Seems to be part of the schema.go.

```suggestion
atmos describe component vpc -s plat-ue2-prod --pager=yes
```

Reply:
`more` is to disable and since pager is default behavior, the intent here was to show that pager argument existed.

having a pager with yes arg does not make sense.